tests/x11/yast2_lan_restart.pm

```diff
@@ -42,7 +42,20 @@ sub check_network {
     $status //= 'no_restart';
     wait_still_screen;
     send_key 'alt-o';                     # OK
-    assert_screen 'yast2_closed_xterm_visible', 120;
+                                          # new: warning pops up for firewall, alt-y for assign it to zone
+    assert_screen([qw(yast2-lan-restart_firewall_active_warning yast2_closed_xterm_visible)], 120);
+    if (match_has_tag 'yast2-lan-restart_firewall_active_warning') {
+        send_key 'alt-y';
+        wait_still_screen;
```

Comment:
That means at least 7 seconds of waiting, better use something else, at least a reduced stilltime